webapp/src/components/createCategory/createCategory.tsx

```diff
@@ -28,7 +28,7 @@ type Props = {
     initialValue?: string
     onClose: () => void
     title: JSX.Element
-    onCreate?: () => Promise<void>
+    onCreate?: (id: Category) => Promise<void>
```

Comment:
the parameter name is misleading, it should be `category: Category`

Reply:
ok thank you! I just fixed it by https://github.com/mattermost/focalboard/pull/4057/commits/59961f68fc964b5eb68dd9cde6232ce6843e6e93